src/router.js

```diff
@@ -17,13 +18,33 @@ const guardTipComments = (to, from, next) => {
   }
 };
 
+const guardCommentView = (to, from, next) => {
+  if (to.name === 'comment'
+      && typeof to.params.id !== 'undefined'
+      && typeof to.params.tipId !== 'undefined') {
+    next();
+  } else {
+    next('/');
```

Comment:
why not 404?

Reply:
We don't have 404 page. I guess this a matter of a different user-story. Our maintenance page is there to show that the site is down.